amgut/templates/taxa.html

```diff
@@ -0,0 +1,236 @@
+{%extends sitebase.html%}
+{% block head %}
+    <script src="/static/vendor/js/Chart.bundle.min.js"></script>
+    <script src="/static/vendor/js/chroma.min.js"></script>
+    <script src="/static/js/interactive.js"></script>
+    <script type="text/javascript">
+      var barChartSummaryData = {
+        labels: {% raw [b['barcode'] + ' - ' + t for b, t in zip(barcodes, titles)] %},
+        rawLabels: null,
+        datasets: null,
+        rawData: {% raw datasets %},
+        metaData: null
+      };
+      var barChartFoldData = {
+        labels: [],
+        datasets: [],
+        barcodes: {% raw [b['barcode'] for b in barcodes] %},
+        titles: {% raw titles %}
+      };
+      var available_summaries = {% raw available_summaries %};
```

Comment:
I think this is over-indented or the rest of the code is under-indented, depending on how you want to see it ...